omod/src/main/webapp/pages/markPatientDead.gsp

```diff
@@ -19,6 +19,24 @@
             link: '${ ui.urlBind("/" + contextPath + "coreapps/clinicianfacing/patient.page?patientId="+patientId, [ patientId: patient ] ) }'
         }
     ];
+
+    jq(function() {
```

Comment:
@slubwama Use the containers layout option for SMC Form - makes this code easier to read and follow